src/main/java/com/google/idea/perf/methodtracer/TracerConfig.kt

```diff
@@ -151,7 +246,33 @@ object TracerConfig {
     fun shouldInstrumentClass(classJvmName: String): Boolean {
         lock.withLock {
             val classConfig = classConfigs[classJvmName] ?: return false
-            return classConfig.methodNames.isNotEmpty() || classConfig.methodIds.isNotEmpty()
+            return classConfig.commands?.isNotEmpty() ?: true || classConfig.methodIds.isNotEmpty()
```

Comment:
Let's add some parentheses here, since it's not obvious which of `?:` or `||` has precedence.